packages/web/src/components/createCell.tsx

```diff
@@ -1,39 +1,43 @@
-import { useQuery, OperationResult } from './GraphQLHooksProvider'
+import { A } from 'ts-toolbelt'
```

Comment:
haha not sure what the change here is peter! But yes its a devDep, did you mean import type only?

Reply:
Done 👍

Reply:
lol, I don't know why I used the suggestion tool... top-kek!

I mean that because it's a devDependency it won't be installed for users, and it looks like a "design time requirement," basically I'm not sure what the type looks like when it's generated? Does it reference `A` or does it interpolate that value and expand it in the generated type def?